tests/unit/common/test_generators.py

```diff
@@ -45,4 +45,25 @@ def test_multivariate_vector_generation_2d(self, mean, cov):
 
 
 class TestConstantGenerator:
-    pass
+    @pytest.mark.parametrize(
+        "constant",
+        [
+            2,
+            10111,
+            (np.array([1]),),
+            (np.array([1, 2]),),
+            (np.array([1, 2, 3]),),
+            (np.array([[1, 1], [1, 1]]),),
+            (np.array([[1, 2, 3], [3, 2, 1]]),),
+        ],
+    )
```

Comment:
Can you add more test cases where we use floating point values, not just integers? Both for scalars and arrays. Also try mixing them as well inside the array (ints and floats).